### PR TITLE
Created my first Goodie trying to display it in a cool way

### PR DESCRIPTION
**Pull request by juarezpaf**
**What does your Instant Answer do?**
Create my first Goodie following the *Goodie Quick Start*

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Just helped me to understand better how to create a Goodie

**What is the data source for your Instant Answer? (Provide a link if possible)**
none

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**
`duckduckhack juarezpaf`

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
me

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes.

**Which existing Instant Answers will this one supercede/overlap with?**
none

**Are you having any problems? Do you need our help with anything?**
Yes. I didn't make the correct test and I don't know the best way to handle html tags inside of perl.

**Where did you hear about DuckDuckHack? (For first time contributors)**
http://remotehackers.com/

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
duckduckhack-juarezpaf

## Checklist
Please place an 'X' where appropriate.

```
[X] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
[] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
[X] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
[] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
[] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 10

    - Windows 7
        [] Google Chrome
        [] Firefox
        [] Opera
        [] IE 8
        [] IE 9
        [] IE 10

    - Windows XP
        [] IE 7
        [] IE 8

    - Mac OSX
        [X] Google Chrome
        [X] Firefox
        [X] Opera
        [X] Safari

    - iOS (iPhone)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - iOS (iPad)
        [] Safari Mobile
        [] Google Chrome
        [] Opera

    - Android
        [] Firefox
        [] Native Browser
        [] Google Chrome
        [] Opera
```

### CODE REVIEW

**Comment by jagtalon** [comment]
@moollaza this looks okay to me as long as it only triggers on that query. CC @moollaza

File: lib/DDG/Goodie/IsAwesome/juarezpaf.pm [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+category "special";
+topics "geek", "special_interest";
+code_url "https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/lib/DDG/Goodie/isAwesome/juarezpaf.pm";
+attribution github => ["https://github.com/juarezpaf", "juarezpaf"],
+            twitter => "juarezpaf";
+triggers start => "duckduckhack juarezpaf";
+
+# Handle statement
+handle remainder => sub {
+	return if $_;
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: Can you fix the indentation here? Should be 4 spaces per tab

**Comment by javathunderman** [comment]
Puts my First Goodie to shame... :)

File: lib/DDG/Goodie/IsAwesome/juarezpaf.pm [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    return html => "<div class='zci_main'>" .
+                   "<div class='zci__img-wrap'><a href='http://twitter.com/juarezpaf'><img class='zci__img' src='https://pbs.twimg.com/profile_images/511615958297747456/dLEI9Ioo.jpeg'></a></div>".
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: This img src should be proxied through `/iu/?u=$url`

**Comment by moollaza** [comment]
@juarezpaf @jagtalon this is certainly an interesting approach to the "First Goodie" tutorial!

I suppose this is okay, given the trigger, but I would really prefer to see this implemented with a Handlebars template (not yet possible, but coming soon!)

Nicely done @juarezpaf

File: lib/DDG/Goodie/IsAwesome/juarezpaf.pm (hunk 2 above)
Comment (jagtalon) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: Good catch, @mintsoft!

Comment (MrChrisW) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: @juarezpaf Just to clarify the image link should be `https://duckduckgo.com/iu/?u=https://pbs.twimg.com/profile_images/511615958297747456/dLEI9Ioo_400x400.jpeg`

File: lib/DDG/Goodie/IsAwesome/juarezpaf.pm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                   "<div class='zci__body'>".
+                   "<h1 class='zci__header has-sub'>Juarez Filho <span class='zci__header__sub'>completed DuckDuckHack Goodie tutorial</span></h1>".
```
Comment (jagtalon) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: Should probably be "completed **the** DuckDuckHack Goodie ..."

File: lib/DDG/Goodie/IsAwesome/juarezpaf.pm [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+                   "<div class='zci__content'>A enthusiastic Front end Engineer with over 7 years of experience, started his career as PHP Developer, he tried to learn Java and after surrendered to Ruby and Rails but with his first CSS class he devoted completly to Front End and UX. Worked with several Startups, including Eadbox, Amanaie, Enjoei and recently Gabstr Inc.</div>".
```
Comment (jagtalon) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: "front-end" instead? The capitalization and lack of hyphen looks kinda weird to me. :) Or maybe "frontend" is good too. Typo with "completely" as well. Probably shouldn't titlecase "Startups".

Comment (jagtalon) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: I like this IA, although this part feels too much like an ad--and we don't want that on the site. I dunno. Please spell check though!

Comment (jagtalon) [inline lib/DDG/Goodie/IsAwesome/juarezpaf.pm]: Should be "An enthusiastic ..." instead of "A enthusiastic ...", too.

**Comment by moollaza** [comment]
ping! Hey @juarezpaf , I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

**Comment by mintsoft** [comment]
:sob:

**Comment by mintsoft** [comment]
@juarezpaf I'm closing this for now; feel free to reopen it whenever you want!

**Pull request closed without merging**